index.html

```diff
@@ -243,7 +248,25 @@ <h3>LET’S CONNECT</h3>
         </div>
       </section>
       <!--End of about section-->
-
-
+       <!--Start of Contact Page-->
+       <section class="contact-page">
+        <div class="contact-block">
+            <div class="description">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!</p>
+            </div>
+            <form action="#">
```

Comment:
- [ ] Nice work implementing the form, it look really good :clap: However it states in the project requirement that data should be collected by using the [Formspree service](https://formspree.io/html). Therefore I suggest you:
  - add the `action` attribute to your form tag and its value should be similar to `https://formspree.io/f/{form_id}` where `{form_id}`  is `id` giving you when create an account on [formspree service](https://formspree.io/html).
  - add the `method` attribute to your `form` tag and its value  equals `POST`